contracts/GovHub.template

```diff
@@ -331,7 +331,7 @@ contract GovHub is System, IParamSubscriber {
       members[addr] = index;
     }
     memberSet.pop();
-    delete members[member];
+    delete members[member]; //@openissue
```

Comment:
shouldn't we revert deleted user's votings on uncompleted proposals as part of his deletion procedure?
this is not trivial - doing nothing is simpler - but achievable.